src/sidebar/store/modules/groups.js

```diff
@@ -62,6 +97,7 @@ const reducers = {
 
   CLEAR_GROUPS() {
     return {
+      filteredGroupIds: null,
```

Comment:
This is new to this updated branch. If we fully clear groups, I think we should clear out filtered groups. It doesn't strictly apply to our use case, but seems wise for the future.